yt_dlp/extractor/crunchyroll.py

```diff
@@ -42,70 +46,73 @@ class CrunchyrollBaseIE(InfoExtractor):
         'ru': 'ru-RU',
         'hi': 'hi-IN',
     }
+    _CF_HINT = 'Request blocked by Cloudflare; navigate to Crunchyroll in your browser, then pass the fresh cookies (with --cookies-from-browser or --cookies) and your browser\'s User-Agent (with --user-agent)'
```

Comment:
I thought curl-cffi works for crunchyroll, no?

Reply:
It does, but since using the cookies/user-agent bypass method is now commonplace among our users, and since curl-cffi is only available in 2 of our builds, I think it would be a regression to remove this hint and only rely on the impersonate API warning.

I think we should utilize the impersonate API in this extractor after #9756 merged, so that the extractor can determine how to handle the 403 error based on whether or not impersonation was used for the request.

Also, if it wasn't clear in the PR description, the 403 error that this PR fixes is completely unrelated to cloudflare/impersonation